components/gfx/font_cache_task.rs

```diff
@@ -198,15 +218,15 @@ impl FontCache {
         for_each_available_family(|family_name| {
             let family_name = LowercaseString::new(&family_name);
             if !self.local_families.contains_key(&family_name) {
```

Comment:
It's only done when the local families are refreshed (which is only on application startup at the moment), so I wouldn't worry about it.